### PR TITLE
doc: added docs to updated resources 2

### PR DESCRIPTION
**Pull request by Adelmar92**
Added documentation in schema resources:

this first PR contans all the resources that have been already updated to the new SDK so the deploy does not get delayed

Validation:
to validate all the resources ive run the next script to excecute the cfn generate in all the modified schemas:
``` bash
#!/bin/bash

# Navigate to the current directory
current_directory=$(pwd)

# Iterate through all subdirectories
for folder in "$current_directory"/*/; do
  if [ -d "$folder" ]; then
    echo "GENERATING ----- ${folder}"
    # Change to the subdirectory
    cd "$folder"

    # Run the command 'cfn generate'
    cfn generate

    # Return to the original directory
    cd "$current_directory"
  fi
done
``` 

_Jira ticket_: [1161](https://jira.mongodb.org/browse/INTMDB-1161)

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Manual QA performed:

- [ ] cfn invoke for each of CRUDL/cfn test
- [ ] Updated resource in  [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples)
- [ ] Published to AWS private registry
- [ ] Used the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create and update a stack in AWS
- [ ] Deleted stack to ensure resources are deleted
- [ ] Created multiple resources in same stack
- [ ] Validated in Atlas UI
- [ ] Included screenshots

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] For CFN Resources: I have released by changes in the private registry and proved by change
  works in Atlas

### CODE REVIEW

**Review by lantoli — APPROVED** [review]
knit: i don't if if would be better to have the url for the resource instead of README, e.g. https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/global-cluster-config instead of https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/global-cluster-config/README.md

**Comment by AgustinBettati** [comment]
> knit: i don't if if would be better to have the url for the resource instead of README, e.g. https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/global-cluster-config instead of https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/global-cluster-config/README.md

Looking into some examples (https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/cloud-backup-restore-jobs/README.md) I see that they reference the README.md directly so we should be good

**Review by AgustinBettati — APPROVED** [review]

File: cfn-resources/access-list-api-key/mongodb-atlas-accesslistapikey.json [hunk 1]
```diff
@@ -1,70 +1,73 @@
 {
-    "typeName": "MongoDB::Atlas::AccessListAPIKey",
-    "description": "Creates the access list entries for the specified organization API key.",
-    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
-    "definitions": {
+  "typeName": "MongoDB::Atlas::AccessListAPIKey",
+  "description": "Creates the access list entries for the specified organization API key.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
```
Comment (AgustinBettati) [inline cfn-resources/access-list-api-key/mongodb-atlas-accesslistapikey.json:4]: Looking into Aastha comment in jira card (https://jira.mongodb.org/browse/INTMDB-1161?focusedCommentId=5741225&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-5741225) we can modify the sourceUrl property in these PRs. What do you think?

**Review by AgustinBettati — CHANGES_REQUESTED** [review]

File: cfn-resources/access-list-api-key/mongodb-atlas-accesslistapikey.json (hunk 1 above)
Comment (Adelmar92) [reply cfn-resources/access-list-api-key/mongodb-atlas-accesslistapikey.json]: done!

**Comment by Adelmar92** [comment]
> > knit: i don't if if would be better to have the url for the resource instead of README, e.g. https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/global-cluster-config instead of https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/global-cluster-config/README.md
> 
> Looking into some examples (https://github.com/aws-quickstart/quickstart-kubernetes-resource-provider/blob/main/get/awsqs-kubernetes-get.json#L5) I see that they reference the README.md directly so we should be good

@lantoli +1 in what Agustin said, is better to link the readme, anyways i also added the sourceURL

**Review by AgustinBettati — APPROVED** [review]
LGTM, thanks for adjusting

**Pull request merged**